custom/icds_reports/static/js/directives/adolescent-girls/adolescent-girls.directive.js

```diff
@@ -3,10 +3,10 @@ var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function AdolescentWomenController($scope, $routeParams, $location, $filter, demographicsService, locationsService,
     dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
-    baseControllersService, isAlertActive, isMobile) {
+    baseControllersService, isAlertActive, isMobile, haveAccessToFeatures) {
```

Comment:
This file canges the Map and Chart tooltips


![ps_chart2](https://user-images.githubusercontent.com/21310920/71279680-65905d00-237f-11ea-8d97-79c428bb245d.png)
![ps_map](https://user-images.githubusercontent.com/21310920/71279687-67f2b700-237f-11ea-967f-03b00f76a2dd.png)